### DISCUSSION TITLE
Help to set up in NAS Synology

### DISCUSSION THREAD

**Discussion by vixius** [jlesage/docker-dupeguru#24] [Q&A]
Hello,

I read many thread but still can't get working.
I installed it in registry by container manager
Tried the script wrote in documentation. and this [guide](https://github.com/jlesage/docker-czkawka/issues/5#issuecomment-1439903987)
Even set group id and user id but when I try to enter by IP it says can't connect

Command:

docker run -d
--name=czkawka
-p 5800:5800
-e USER_ID=1026
-e GROUP_ID=101
-v /volume1/docker/czkawka/config:/config:rw
-v </volume1/homes/Vix/Photos>:/storage/photos:rw
jlesage/czkawka

I'm stuck. Please help am getting crazy with that

**Comment by jlesage** [comment]
I guess it should be `-v /volume1/homes/Vix/Photos:/storage/photos:rw` instead of `-v </volume1/homes/Vix/Photos>:/storage/photos:rw` (no `<` `>` characters) ?

**Comment by vixius** [comment]
thanks, I finally got working. It was to set root instead of admin.

**Comment by Christian72D** [comment]
I know, it´s solved.
But for me, THIS explanation works best for me.

[https://medium.com/@stanislavalexandrov/running-dupeguru-on-synology-with-docker-quic-5a666fdb80f4](url)
